mp/src/game/shared/momentum/weapon/weapon_mom_stickybomblauncher.cpp

```diff
@@ -278,6 +286,15 @@ float CMomentumStickybombLauncher::CalculateProjectileSpeed(float flProgress)
                            MOM_STICKYBOMB_MIN_CHARGE_VEL, MOM_STICKYBOMB_MAX_CHARGE_VEL);
 }
 
+void CMomentumStickybombLauncher::StopChargeSound() 
+{
+    if (m_flChargeBeginTime > 0) // stop only if charging
+    {
+        StopWeaponSound(GetWeaponSound("charge"));
```

Comment:
I was saying regardless of charge time, you can stop the sound. This can move out of the if check.